x-pack/plugins/maps/public/classes/styles/vector/components/symbol/icon_select.js

```diff
@@ -95,26 +128,66 @@ export class IconSelect extends Component {
   }
 
   _renderIconSelectable() {
-    const options = SYMBOL_OPTIONS.map(({ value, label }) => {
-      return {
-        value,
-        label,
-        prepend: (
-          <SymbolIcon
-            key={value}
-            symbolId={value}
-            fill={getIsDarkMode() ? 'rgb(223, 229, 239)' : 'rgb(52, 55, 65)'}
-          />
-        ),
-      };
-    });
+    const makiOptions = [
+      {
+        label: 'Maki icons',
```

Comment:
Needs to be i18n string, also change to "Kibana icons" or "Provided icons". We should not leak "maki" implementation detail to users.